drivers/gles3/rasterizer_scene_gles3.h

```diff
@@ -124,6 +124,7 @@ class RasterizerSceneGLES3 : public RasterizerScene {
 			float z_slope_scale;
 			float shadow_dual_paraboloid_render_zfar;
 			float shadow_dual_paraboloid_render_side;
+			float viewport_size[2];
```

Comment:
by adding this, you may need to remove the padding at the bottom. If i understood well, these structs need to be padded to 16 bytes or they wont work in webgl